server/handlers/policy_relationship_handler.go

```diff
@@ -37,8 +41,45 @@ func (h *Handler) GetRegoPolicyForDesignFile(
 		return
 	}
 
+	name := mux.Vars(r)["name"]
```

Comment:
@theBeginner86 How calling the policies from this modified endpoint looks like from client?